### ISSUE TITLE
error 500 when uploading problem zip 

### ISSUE THREAD

**Issue by Gusser93** [DOMjudge/domjudge#583]
When uploading the problem zip [coins.zip](https://github.com/DOMjudge/domjudge-packaging/files/3142050/coins.zip)  I get an "Internal Server Error"

symfony-log says
```
[2019-05-03 15:15:17] php.WARNING: Notice: Undefined variable: zip {"exception":"[object] (ErrorException(code: 0): Notice: Undefined variable: zip at /opt/domjudge/domserver/webapp/src/DOMJudgeBundle/Controller/Jury/ProblemController.php:131)"} []
[2019-05-03 15:15:17] php.CRITICAL: Call to a member function close() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function close() on null at /opt/domjudge/domserver/webapp/src/DOMJudgeBundle/Controller/Jury/ProblemController.php:131, Symfony\\Component\\HttpKernel\\Exception\\ServiceUnavailableHttpException(code: 0): No valid zip archive given at /opt/domjudge/domserver/webapp/src/DOMJudgeBundle/Service/DOMJudgeService.php:590)"} []
[2019-05-03 15:15:17] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function close() on null" at /opt/domjudge/domserver/webapp/src/DOMJudgeBundle/Controller/Jury/ProblemController.php line 131 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function close() on null at /opt/domjudge/domserver/webapp/src/DOMJudgeBundle/Controller/Jury/ProblemController.php:131, Symfony\\Component\\HttpKernel\\Exception\\ServiceUnavailableHttpException(code: 0): No valid zip archive given at /opt/domjudge/domserver/webapp/src/DOMJudgeBundle/Service/DOMJudgeService.php:590)"} []
```

DOMjudge: | 7.0.0
-- | --
Environment: | prod
Debug: | No
 
PHP: | 7.0.33-0+deb9u3
Symfony: | 3.4.26
Twig: | 2.7.4

Installed with ReadMe from DockerHub. 
Tested with Firefox 67.0b15 (64-Bit) and Chromium 74.0.3729.131

**Comment by nickygerritsen** [comment]
The problem seemed to be that PHP's `ZipArchive` doesn't like your zip. We get an `ZIPARCHIVE::ER_INCONS` error while opening it, which we did not display correctly.

I have fixed that we now display that error correctly, but the ZIP will still not import. I', not entirely sure what is 'inconsistent' about your zip, but if I use the commandline `unzip` and `zip` command to extract and zip the files again, it works.
